### ISSUE TITLE
K3D Destroy panic

### ISSUE THREAD

**Issue by boskiv** [kubefirst/kubefirst#1756]
### Which version of kubefirst are you using?

v2.2.9

### Which cloud provider?

k3d (local)

### Which installation type?

CLI

### Which distributed Git provider?

GitHub

### What is the issue?

```
❯ kubefirst k3d destroy
------------------------------------------------
Follow your logs in a new terminal with:
   tail -f -n +1 /Users/iskiridomov/.k1/logs/log_1692364874.log
------------------------------------------------
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x28 pc=0x103fb53ec]

goroutine 1 [running]:
github.com/kubefirst/runtime/pkg/k8s.CheckForExistingPortForwards({0x14000b8f688, 0x1, 0x14000b8f5a8?})
	github.com/kubefirst/runtime@v0.3.4/pkg/k8s/helpers.go:19 +0xac
github.com/kubefirst/kubefirst/cmd/k3d.destroyK3d(0x14000e0d400?, {0x1049690dd?, 0x4?, 0x104968fd9?})
	github.com/kubefirst/kubefirst/cmd/k3d/destroy.go:40 +0x9c
github.com/spf13/cobra.(*Command).execute(0x14000c1cf00, {0x1075e2b20, 0x0, 0x0})
	github.com/spf13/cobra@v1.7.0/command.go:940 +0x658
github.com/spf13/cobra.(*Command).ExecuteC(0x107577400)
	github.com/spf13/cobra@v1.7.0/command.go:1068 +0x320
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.7.0/command.go:992
github.com/kubefirst/kubefirst/cmd.Execute()
	github.com/kubefirst/kubefirst/cmd/root.go:48 +0x2c
main.main()
	github.com/kubefirst/kubefirst/main.go:87 +0x59c
❯ tail -f -n +1 /Users/iskiridomov/.k1/logs/log_1692364874.log
2023-08-18T16:21 INF pkg/helpers.go:78 > Using config file: /Users/iskiridomov/.kubefirst
❯ k3d cluster list
NAME         SERVERS   AGENTS   LOADBALANCER
backoffice   0/3       0/3      true
kubefirst    1/1       3/3      true
❯ kubefirst version
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by boskiv** [comment]
I had this screen running if it's means
![Screenshot 2023-08-18 at 6 00 12 PM](https://github.com/kubefirst/kubefirst/assets/1939783/09631fca-092b-49ce-ab03-402970638dc6)
After I quit it, `kubefirst k3d destroy` executed normally

**Comment by fharper** [comment]
@boskiv sorry for the late reply.

This should be fixed in [v2.2.8](https://github.com/kubefirst/kubefirst/releases/tag/v2.2.8) (check https://github.com/kubefirst/kubefirst/issues/1684), but you were using v2.2.9 which is weird.

I can't reproduce with the latest version v2.2.17

![CleanShot 2023-09-13 at 17 21 45@2x](https://github.com/kubefirst/kubefirst/assets/1170989/5cde5331-3727-404d-95ba-cf8d9a5111f0)

Can you give it another try, and see if you can reproduce with the latest version of kubefirst please?

**Comment by fharper** [comment]
Since I had no news, I'll close the issue. If you can still reproduce with the latest version of kubefirst, we can reopen the issue, just let us know please.
